Makefile

```diff
@@ -279,6 +284,17 @@ manifest:
 	$(CURDIR)/hack/generate-manifest-octant.sh --mode dev > build/yamls/antrea-octant.yml
 	$(CURDIR)/hack/generate-manifest-windows.sh --mode dev > build/yamls/antrea-windows.yml
 
+.PHONY: scale-manifest
+scale-manifest:
+	@echo "===> Generating dev manifest for Antrea <==="
+	$(CURDIR)/hack/generate-manifest.sh --mode dev --simulator > build/yamls/antrea.yml
+	$(CURDIR)/hack/generate-manifest.sh --mode dev --ipsec > build/yamls/antrea-ipsec.yml
```

Comment:
Agree. Since we actually don't release the yaml, I also think we don't need a target here.

Reply:
@tnqn @weiqiangt do you think it is better to add the yaml file than generating yaml file by scripts?

Reply:
IMO, I actually prefer the opposite way. We can just generate one once we're going to use the simulator YAML.
But one quick question, how can users tune the number of simulated nodes? Maybe when generating the simulator YAML, the generator should accept an argument to tune the number.

Reply:
The published yaml are for user consumption, I think we don't put yaml for tests there, otherwise there will be too many, like the one for code coverage.

Reply:
> IMO, I actually prefer the opposite way. We can just generate one once we're going to use the simulator YAML.
> But one quick question, how can users tune the number of simulated nodes? Maybe when generating the simulator YAML, the generator should accept an argument to tune the number.

@liu4480, do we have a way to tune the simulator replica number?

Reply:
@weiqiangt I 've proposed a solution, for example:
 make scale-manifest REPLICA=10
this will modify the replica to 10 for simulator